code/modules/mob/living/living_defense.dm

```diff
@@ -323,6 +323,8 @@
 
 
 /mob/living/ratvar_act()
+	if(is_servant_of_ratvar(src))

```

Comment:
Just change the below check to `if(!is_servant_of_ratvar(src) && !add_servant_of_ratvar(src))`